<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalProgramsEditor</class>
 <widget class="QDialog" name="PersonalProgramsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Индивидуальные программы</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="tvPrograms">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QFrame" name="frProgramParams">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gbPP">
         <property name="title">
          <string>Параметры индивидуальной программы</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Название</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edName"/>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="gbIcon">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>Логотип</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lblLogo">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnLogoSelect">
               <property name="text">
                <string>Выбрать...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblMinTimeBetweenDP">
            <property name="text">
             <string>Минимальное время между дневными программми</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbMinTimeBetweenDP"/>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblMaxTimeBetweenDP">
            <property name="text">
             <string>Максимальное время между дневными программми</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbMaxTimeBetweenDP"/>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="gbSchedule">
            <property name="title">
             <string>Расписание</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QTableView" name="tvSchedule">
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>300</number>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frScheduleButtons">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QPushButton" name="btnPPAddDP">
                  <property name="toolTip">
                   <string>Добавить дневную программу в расписание...</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../kernel.qrc">
                    <normaloff>:/images/Plus_Easy.png</normaloff>:/images/Plus_Easy.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPPDoubleDP">
                  <property name="toolTip">
                   <string>Дублировать дневную программу</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../kernel.qrc">
                    <normaloff>:/images/Double.png</normaloff>:/images/Double.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPPDelDP">
                  <property name="toolTip">
                   <string>Удалить дневную программу из расписания...</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../kernel.qrc">
                    <normaloff>:/images/Close.png</normaloff>:/images/Close.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPPDPUp">
                  <property name="toolTip">
                   <string>Переместить дневную программу на один шаг к началу расписания</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../kernel.qrc">
                    <normaloff>:/images/Up_Easy.png</normaloff>:/images/Up_Easy.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPPDPDn">
                  <property name="toolTip">
                   <string>Переместить дневную программу на один шаг к концу расписания</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../kernel.qrc">
                    <normaloff>:/images/Down_Easy.png</normaloff>:/images/Down_Easy.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frMainButtons">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnAddDP">
            <property name="toolTip">
             <string>Добавить индивидуальную программу</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../kernel.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditDP">
            <property name="toolTip">
             <string>Изменить индивидуальную программу</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../kernel.qrc">
              <normaloff>:/images/MethodsEdit.png</normaloff>:/images/MethodsEdit.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelDP">
            <property name="toolTip">
             <string>Удалить индивидуальную программу...</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../kernel.qrc">
              <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnPPAddDP</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_addDP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPPDelDP</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_delDP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPPDPUp</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_moveDPUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPPDPDn</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_moveDPDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddDP</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_ppAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEditDP</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_ppEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelDP</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_ppDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tvPrograms</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_selectPP(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tvPrograms</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_selectPP(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMinTimeBetweenDP</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_changeMinTime(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMaxTimeBetweenDP</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_changeMaxTime(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLogoSelect</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_selectLogo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPPDoubleDP</sender>
   <signal>clicked()</signal>
   <receiver>PersonalProgramsEditor</receiver>
   <slot>on_doubleDP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_addDP()</slot>
  <slot>on_delDP()</slot>
  <slot>on_moveDPUp()</slot>
  <slot>on_moveDPDown()</slot>
  <slot>on_ppAdd()</slot>
  <slot>on_ppEdit()</slot>
  <slot>on_ppDel()</slot>
  <slot>on_selectPP(QModelIndex)</slot>
  <slot>on_changeMinTime(int)</slot>
  <slot>on_changeMaxTime(int)</slot>
  <slot>on_selectLogo()</slot>
  <slot>on_splitterMoved(int,int)</slot>
  <slot>on_doubleDP()</slot>
 </slots>
</ui>
